gfreshii-backend/src/main/java/com/edbootcamp/dao/IngredientDAOImpl.java

```diff
@@ -0,0 +1,91 @@
+package com.edbootcamp.dao;
+
+import java.util.List;
+
+import javax.transaction.Transactional;
+
+import org.hibernate.Criteria;
+import org.hibernate.Query;
+import org.hibernate.SQLQuery;
+import org.hibernate.Session;
+import org.hibernate.SessionFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.orm.hibernate4.HibernateTemplate;
+import org.springframework.stereotype.Repository;
+import org.springframework.transaction.annotation.EnableTransactionManagement;
+
+import com.edbootcamp.api.dao.IngredientDAO;
+import com.edbootcamp.api.entity.Ingredient;
+import com.edbootcamp.api.entity.Recipe;
+import com.edbootcamp.entity.IngredientImpl;
+import com.edbootcamp.entity.RecipeImpl;
+
+@Repository
+@EnableTransactionManagement
+@Transactional
+public class IngredientDAOImpl implements IngredientDAO{
+
+	@Autowired
+	private SessionFactory sessionFactory;
+	
+	@Autowired 
+	HibernateTemplate hibernateTemplate;
+	
+	@Override
+	public List<Ingredient> allIngredients(Long id) {
+		Session session = sessionFactory.getCurrentSession();
+		Query query = session.createQuery("from IngredientImpl where recipeID=:id");
```

Comment:
Minor feedback on style - Typically SQL queries are in all caps, so "FROM", "WHERE", "DELETE", etc.
And then similarly here I would think you need a "SELECT" in front of the "FROM"